### DISCUSSION TITLE
owlculus - installation issues

### DISCUSSION THREAD

**Discussion by Caelst** [be0vlk/owlculus#1] [Q&A]
First off, I really appreciate the idea/project you have put together here. 
As the title suggests, I am having issues with executing this install. Not one for using the forms, I have spent the past few days working out my methods and trying to discover how I am going wrong.

Installed fresh Ubuntu 22.04 on physical at home
Installed updates
Installed python-latest
Installed mariadb-latest
created user & database for owlculus
Installed nano to edit .env.example and then save as .env.
Installed git - cloned owlculus.git to /owlculus
Installed phpmyadmin - this will allow a partner a GUI/visibility of the database
Ran your scripts for app install - nice touch :)
OSError: cannot load library 'pango-1.0-0' came up - installed manually
Error with dotenv loading came up - installed manually
.env file rights issue came up - chmod 777 the file
Execute "python3 app.py" and no errors returned. 

WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5000
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 821-889-213

Go to the URL and there is nothing. 

This is the same issue I got 1st time around. Decided to write over the OS installed fresh. 
Same results though...

Any thoughts on this?
Something I am missing clearly, I am just hitting my head on the desk now...

**Comment by be0vlk** [comment]
Hey there sorry for the very late response I never got a notification for this! You did run the db_setup script? Any errors showing in the console with debug mode on?

You can also try directly browsing to /cases and see if it properly redirects you.
